pkgs/tools/system/auto-epp/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, pkgs, lib , fetchFromGitHub }:
```

Comment:
Arguments have a recommended order: https://nixos.org/manual/nixpkgs/stable/#sec-syntax (last point)

```suggestion
{ lib, stdenv, fetchFromGitHub, python3 }:
```

Reply:
While you already changed this, I thought I'd just clarify that I learnt the other day that this is no longer the case. I was looking at the stable manual, but the unstable manual (and now part of the CONTRIBUTING.md docs), this recommendation has been explicitly removed